app/models/program.rb

```diff
@@ -113,4 +118,23 @@ def create_difficulty_levels
                                                 color: '#EF6E69')
     true
   end
+
+  ##
+  # Check if languages string has the right format. Used as validation.
+  #
+  def check_languages_format
+    # All white spaces are removed to allow languages to be separated by ',' and ', ' but only the first option is saved
```

Comment:
I mean that `en, de` and `en,de` are both allowed, but it is always saved as `en,de`, removing spaces (I did it like that for being consistent). I will change the comment to make it more clear.